experimental/origin-graphql/src/typeDefs/Growth.js

```diff
@@ -27,16 +27,28 @@ module.exports =
   }
 
   enum GrowthActionType {
-    basicProfile
-    attestations
+    email
+    phoneNumber
+    twitter
+    airbnb
+    facebook
     referral
+    profile
+    listingCreated
+    listingPurchased
   }
 
   enum GrowthInviteStatus {
     pending
     successful
   }
 
+  enum Eligibility {
+    Eligible
```

Comment:
supernit: shoudl this be lowercap, similarily to the other enums above ?

Reply:
good point, changing to lower case